src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -1025,7 +1025,23 @@ public int getBoardCostLowerBound() {
             return walkBoardCost;
         return bikeBoardCost;
     }
-    
+
+    /**
+     * @return The time it actually takes to board a vehicle. Could be significant eg. on airplanes and ferries
+     */
+    public int getBoardTime(TraverseMode transitMode) {
+        Integer i = this.rctx.graph.boardTimes.get(transitMode);
```

Comment:
Without `boardTimes` set in config file, this will throw a `java.lang.NullPointerException` and fail the plan request.